layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -158,7 +153,7 @@
                     "description": "Path to use when creating the capture file. Default is: gfxrecon_capture.gfxr",
                     "type": "SAVE_FILE",
                     "filter": "*.gfxr",
-                    "default": "${LOCAL}/gfxrecon_capture.gfxr",
+                    "default": "",
```

Comment:
This seems incorrect, given that the default value is given in the description above?

Reply:
I think the previous value may not have been a valid value and was just for illustration.  `gfxreconstruct` doesn't parse out `LOCAL` in its processing of the filename, as far as I can tell.  unless the loader itself or vkconfig knows what to do with `LOCAL`.  Maybe the default should be `""` and we should add a note that an empty path is replaced with `gfxrecon_capture...gfxr` in the CWD.

Reply:
vkconfig replaces `${LOCAL}` with the working directory. In general, I think the `"default"` value in the json is used by vkconfig and the layer has its own default if nothing is provided.

Reply:
Sounds like, from @lunarpapillo and @mikes-lunarg ,that it should be returned to `${LOCAL}/gfxrecon_capture.gfxr`.

Reply:
I changed it to "" because vkconfig translates "${LOCAL}/gfxrecon_capture.gfxr" to "/home/david/VulkanSDK/gfxrecon_capture.gfxr". And I'm using vkconfig to create new, updated versions of vk_layer_settings.txt (with some missing stuff getting put in), although we're planning to remove those files in the future. It seems that either "" or the default value of "gfxrecon_capture.gfxr" should be what is used. Thoughts?

Reply:
Would a default value of "gfxrecon_capture.gfxr" mean "$(CWD)/gfxrecon_capture.gfxr"?

Reply:
Changed to "gfxrecon_capture.gfxr"